Marlin/src/lcd/dogm/dogm_Statusscreen.h

```diff
@@ -1367,14 +1367,18 @@
 #endif
 #if STATUS_LOGO_WIDTH
   #ifndef STATUS_LOGO_X
-    #define STATUS_LOGO_X 0
-  #endif
-  #ifndef STATUS_LOGO_Y
-    #define STATUS_LOGO_Y _MIN(0U, (20 - (STATUS_LOGO_HEIGHT)) / 2)
+    #ifdef STATUS_HEATERS_X
+      #define STATUS_LOGO_X (STATUS_HEATERS_X - (STATUS_LOGO_WIDTH) - 1) / 2
+    #else
+      #define STATUS_LOGO_X 0
+    #endif
   #endif
   #ifndef STATUS_LOGO_HEIGHT
     #define STATUS_LOGO_HEIGHT (sizeof(status_logo_bmp) / (STATUS_LOGO_BYTEWIDTH))
   #endif
+  #ifndef STATUS_LOGO_Y
+    #define STATUS_LOGO_Y _MAX(0L, (28L - _MIN(28L, STATUS_LOGO_HEIGHT)) / 2L))
```

Comment:
@thinkyhead yes, looks better.
@sjasonsmith those constants (14,18 in current revision) are based on the max area for the logo. There is no nice constant defined for that yet, but border of the xyz frame is drawn at Y=29 or 30... So height = 28 pixels is max for status screen logo.